validator/main.py

```diff
@@ -328,9 +313,17 @@ def validate(
             found_topics = self.get_topics_llm(value, all_topics)
         # Zero Shot Classifier Only
         elif not self._disable_classifier and self._disable_llm:
-            found_topics = self.get_topics_zero_shot(value, all_topics)
+            found_topics = self._inference(value, all_topics)
         else:
             raise ValueError("Either classifier or llm must be enabled.")
+        
+        model_input = {
+            "text": value,
+            "valid_topics": self._valid_topics,
+            "invalid_topics": self._invalid_topics
+        }
```

Comment:
This needs to move up so it can be sent to the `_inference` call inside the elif block